chapter_attention-mechanisms/index_vn.md

```diff
@@ -17,15 +17,24 @@ This paper declares that when perceiving a stimulus, features are registered ear
 The theory has been one of the most influential psychological models of human visual attention.
 -->
 
-*dịch đoạn phía trên*
+Nghiên cứu về sự tập tập trung là một lĩnh vực rộng lớn với chiều dài lịch sử trong ngành thần kinh học nhận thức.
+Bản chất của thuyết tập trung là sự tập trung vào một tiêu điểm hay sự tập trung của nhận thức, điều này cho phép chúng ta ưu tiên tri giác của mình để giải quyết các công việc một cách hiệu quả. <!-- Nguyên bản: Focalization, concentration of consciousness are. Hơi confused chỗ này "to be" thì chia số nhiều mà lại dùng dấy phảy? Không hiểu hai cụm danh từ đó tương đương nhau hay khác nhau nữa :-( Reviewer consider đoạn này nhé ;) -->
```

Comment:
```suggestion
Trọng tâm của lĩnh vực là sự tập trung nhận thức vào một vài đặc điểm nào đó, điều này cho phép chúng ta ưu tiên tri giác của mình để giải quyết các công việc một cách hiệu quả.
```
chắc dấu phẩy tương đương với `and`: `Focalization and concentration of consciousness are...`. Btw, câu này cả `focalization`, `concentration` và `attention` sang tiếng Việt đều là `tập trung, chú ý`, nên dịch chay rất khó hiểu. Em nghĩ nên giải thích nôm na thôi.